packages/babel-plugin-polyfill-corejs2/src/index.ts

```diff
@@ -22,7 +24,7 @@ type Options = {
     entryInjectRegenerator: boolean;
   };
   "#__secret_key__@babel/runtime__compatibility": void | {
-    useBabelRuntime: string;
+    useBabelRuntime: boolean;
```

Comment:
Is this a breaking change? If so this PR should be landed in a new major.

Reply:
Yes, but these packages are still at 0.x (so breaking changes are in minor releases)

Reply:
Yes, 0.x -> 0.x+1 is what I mean for a new major.